README.md

```diff
@@ -29,7 +29,7 @@ Under .NET Core, [remember to register](https://github.com/nlog/nlog/wiki/Regist
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   
   <extensions>
-    <add assembly="CorentinAltepe.NLog.Loki" />
```

Comment:
Hello. I'm not certain if it should be `assembly="CorentinAltepe.NLog.Loki"` or `assembly="NLog.Loki"` here.

The code's namespace is `NLog.Loki`, but since a Nuget package had already been published with this name and later unlisted (original source code from [anton-gogolev/nlog.loki](https://github.com/anton-gogolev/nlog.loki), this is where I forked this repository from), I had to publish the package under a different name, which I thought would be the assembly name. Hence I choose `CorentinAltepe.NLog.Loki`.

Now, have you tried running the nuget package with assembly name `CorentinAltepe.NLog.Loki` and did it not work?

Reply:
I've verified the published nuget package, and the assembly's name is indeed `NLog.Loki`. I'm confused but it seems your change was the correct one.

Reply:
Yes, that was intentional